apis/appcat/v1/appcat_types.go

```diff
@@ -50,9 +53,25 @@ type AppCatList struct {
 	Items []AppCat `json:"items"`
 }
 
-// AppCatSpec defines the desired state of AppCat
-// The desired state of AppCat is dynamically populated from composition's labels
-type AppCatSpec map[string]string
+// Details are fields that are dynamically parsed from the annotations on a composition.
+type Details map[string]string
```

Comment:
Why the name of this field has been changed?

Reply:
Because there's no more spec field and calling it spec now is just wrong.